cabal.project

```diff
@@ -54,282 +66,10 @@ package cryptonite
   -- generation is dubious. Set the flag so we use /dev/urandom by default.
   flags: -support_rdrand
 
--- ---------------------------------------------------------
--- Disable all tests by default
-
-tests: False
+tests: True
 
 test-show-details: direct
 
--- Then enable specific tests in this repo
-
-package cardano-api
-  tests: True
-
-package cardano-cli
-  tests: True
-
-package cardano-node
-  tests: True
-
-package cardano-node-chairman
-  tests: True
-
-package cardano-submit-api
-  tests: True
-
-package cardano-testnet
-  tests: True
-
-package trace-resources
-  tests: True
-
-package trace-dispatcher
-  tests: True
-
-package trace-forward
-  tests: True
-
-package cardano-tracer
-  tests: True
-
-package trace-resources
-  tests: True
-
-package locli
-  tests: True
-
--- The following is needed because Nix is doing something crazy.
-package byron-spec-ledger
-  tests: False
-
-package vector-map
-  ghc-options: -Werror
-
-package iohk-monitoring
-  tests: False
-
-package ouroboros-consensus-test
-  tests: False
-
-package ouroboros-consensus-cardano-test
-  tests: False
-
-package ouroboros-network
-  tests: False
-
-package ouroboros-network-framework
-  tests: False
-
-package plutus-tx
-  tests: False
-
-package prettyprinter-configurable
-  tests: False
-
-package small-steps
-  tests: False
-
-package small-steps-test
-  tests: False
-
-package tx-generator
-  tests: True
-
-package goblins
-  tests: False
-
-package io-classes
-  tests: False
-
-package cardano-ledger-alonzo-test
-  tests: False
-
--- ---------------------------------------------------------
-
--- The two following one-liners will cut off / restore the remainder of this file (for nix-shell users):
--- when using the "cabal" wrapper script provided by nix-shell.
--- --------------------------- 8< --------------------------
--- Please do not put any `source-repository-package` clause above this line.
-
--- Using a fork until our patches can be merged upstream
-source-repository-package
-  type: git
-  location: https://github.com/input-output-hk/optparse-applicative
-  tag: 7497a29cb998721a9068d5725d49461f2bba0e7a
-  --sha256: 1gvsrg925vynwgqwplgjmp53vj953qyh3wbdf34pw21c8r47w35r
-
-source-repository-package
```

Comment:
This one should remain a `source-repository-package`, it's a fork of a Hackage package, and I'm not sure it meets our criteria for being patched in the repo.

Reply:
We should also remove it from the repo.

Reply:
:+1: